src/audio/audio.cpp

```diff
@@ -251,6 +259,8 @@ bool Audio::initInput(QString inDevDescr)
         return false;
     }
 
+    privateSetGain(Settings::getInstance().getInVolume());
```

Comment:
Yes, that's totally ok. I guess it makes sense adding a (probably customizable) `minGain/maxGain` values to Audio API, defaulting to +-30dB.